types/argon2-browser/argon2-browser-tests.ts

```diff
@@ -1,20 +1,34 @@
-import { argon2 } from 'argon2-browser';
+import 'argon2-browser';
 
-const mandatoryOptions = {
+const hashMandatoryOptions: argon2.HashOptions = {
+  pass: 'Qwerty12?',
+  salt: 'Salty',
+};
+
+const verifyMandatoryOptions: argon2.VerifyOptions = {
+  encoded: '$argon2d$v=19$m=1024,t=1,p=1$U2FsdHlTYWx0eQ$1jpGIXoUTdOQSUFaAP8qCnsr8yTGbF2srlkfrUM+hIo',
   pass: 'Qwerty12?',
-  salt: 'Salty'
 };
 
 (async () => {
-  (await argon2.hash(mandatoryOptions)).encoded; // string
-  (await argon2.hash(mandatoryOptions)).hash; // Uint8Array
-  (await argon2.hash(mandatoryOptions)).hashHex; // string
+  (await argon2.hash(hashMandatoryOptions)).encoded; // string
+  (await argon2.hash(hashMandatoryOptions)).hash; // Uint8Array
+  (await argon2.hash(hashMandatoryOptions)).hashHex; // string
+
+  (await argon2.hash({ ...hashMandatoryOptions, distPath: 'path' })).encoded; // string
+  (await argon2.hash({ ...hashMandatoryOptions, hashLen: 24 })).encoded; // string
+  (await argon2.hash({ ...hashMandatoryOptions, mem: 1024 })).encoded; // string
+  (await argon2.hash({ ...hashMandatoryOptions, parallelism: 1 })).encoded; // string
+  (await argon2.hash({ ...hashMandatoryOptions, time: 1 })).encoded; // string
+  (await argon2.hash({ ...hashMandatoryOptions, type: argon2.ArgonType.Argon2d })).encoded; // string
+  (await argon2.hash({ ...hashMandatoryOptions, type: argon2.ArgonType.Argon2i })).encoded; // string
+  (await argon2.hash({ ...hashMandatoryOptions, type: argon2.ArgonType.Argon2id })).encoded; // string
+
+  await argon2.verify(verifyMandatoryOptions);
```

Comment:
`verify` actually returns a `Promise<void>` and not `Promise<{}>`. Do you think it's worth to `$ExpectType void` ?

Reply:
Sorry @gilbsgilbs to reply so late! Did you test `$ExpectType void` with a `dtslint .` inside the directory? Because on my side, it doesn't return `void` but `{}` unfortunately:

```
ERROR: 29:3  expect  Expected type to be:
  void
got:
  {}
```

Reply:
No worries. None of us are getting paid for this :wink: .

Actually, I did test and I did not have the error. I think you're based on an old version of my branch because I used to think that `verify` promise did resolve to `{}`. Sorry about that.

I added the `void` type assertion in a new commit anyway so that CI can settle the debate.

Reply:
Oh so `void` is fine with me :) I approve.